ethstore/src/lib.rs

```diff
@@ -35,6 +35,7 @@ extern crate ethcore_bigint as bigint;
 extern crate ethcrypto as crypto;
 extern crate ethkey as _ethkey;
 extern crate parity_wordlist;
+extern crate ethcore_util as util;
```

Comment:
I can't see where it's used. Isn't it redundant? Besides that `ethstore` is also compiled for android and ios. Bringing `util` dependency here, breaks that.